AutomatticTracks/lib/src/main/java/com/automattic/android/tracks/crashlogging/internal/SentryErrorTrackerWrapper.kt

```diff
@@ -0,0 +1,41 @@
+package com.automattic.android.tracks.crashlogging.internal
+
+import android.content.Context
+import io.sentry.Sentry
+import io.sentry.SentryEvent
+import io.sentry.SentryOptions
+import io.sentry.android.core.SentryAndroid
+import io.sentry.protocol.User
+
+internal class SentryErrorTrackerWrapper {
```

Comment:
This file is not changed - it's just moved (I'm not sure why git treats this file differently)